### ISSUE TITLE
What about list items without text, only code blocks?

### ISSUE THREAD

**Issue by h0uter** [rvben/rumdl#288]
Currently a list item with only a code block (no text) is flagged by RUMDL: `Spaces after list markers (Expected: 1; Actual: 0)rumdl[MD030](https://github.com/rvben/rumdl/blob/main/docs/md030.md)`.
But when the space is added it is flagged again as a trailing space.

Is this intended behavior? 

# Example

```markdown
- Below is a bullet code block with text

    ```python
    print("hello World")
    ```
- Below is a bullet code block only (no text)
-
    ```python
    print("bullet code block only (no text)")
    ```
- Hello Again
  - Below is a sub bullet code block with text
      ```python
      print("sub bullet code block with text")
      ```
  - Below is a sub bullet code block only (no text)
  - 
      ```python
      print("sub bullet code block only (no text)")
      ```
```

## Renders as

- Below is a bullet code block with text

    ```python
    print("hello World")
    ```
- Below is a bullet code block only (no text)
-
    ```python
    print("bullet code block only (no text)")
    ```
- Hello Again
  - Below is a sub bullet code block with text
      ```python
      print("sub bullet code block with text")
      ```
  - Below is a sub bullet code block only (no text)
  - 
      ```python
      print("sub bullet code block only (no text)")
      ```

**Comment by rvben** [comment]
Thanks for raising this, @h0uter!

Fixed in v0.0.216.

MD030 now skips empty marker lines (list markers followed by nothing or just whitespace). This avoids the conflict between "add a space" and "trailing space" warnings.
